data-processing-lib/ray/src/data_processing_ray/runtime/ray/ray_utils.py

```diff
@@ -113,7 +113,7 @@ def operator() -> ActorHandle:
         actors = [operator() for _ in range(n_actors)]
         for i in range(120):
             time.sleep(1)
-            alive = list_actors(filters=[("class_name", "=", cls_name), ("state", "=", "ALIVE")])
+            alive = list_actors(filters=[("class_name", "=", cls_name), ("state", "=", "ALIVE")], limit=1000)
```

Comment:
list_actors() is now in a different package.  See https://docs.ray.io/en/latest/ray-observability/reference/doc/ray.util.state.list_actors.html.  Should we be using that one?

Reply:
Its version dependent